tedana/workflows/tedana.py

```diff
@@ -261,11 +262,11 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
         which is slightly slower but may be more accurate.
     verbose : :obj:`bool`, optional
         Generate intermediate and additional files. Default is False.
-    png : obj:'bool', optional
-        Generate simple plots and figures. Default is false.
+    nopng : obj:'bool', optional
```

Comment:
I'm not sure it's worth exposing this -- it'd make sense to me to generate figures by default. WDYT ?

Reply:
I think this is fixed in f6cba2e